editor/editor_node.cpp

```diff
@@ -2806,7 +2806,9 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 			OS::get_singleton()->shell_open("https://godotengine.org/qa/");
 		} break;
 		case HELP_REPORT_A_BUG: {
-			OS::get_singleton()->shell_open("https://github.com/godotengine/godot/issues");
+			String system_info = OS::get_singleton()->get_name();
```

Comment:
I would like to add video driver and graphics card information here, however that functionality is still missing since Vulkan was added. Additionally Graphics card information is marked as private under the Vulkan driver - not sure if there was a specific reason for that. It would be nice to have.

Reply:
`RenderingServer::get_singleton()->get_video_adapter_name()` should do the job. Feel free to make this method public if it's not already, but it's already exposed to scripting at least.

Reply:
Thank you! I didn't know that existed.